src/js/messaging/components/MessageSearch.jsx

```diff
@@ -40,25 +40,33 @@ class MessageSearch extends React.Component {
   }
 
   render() {
+    const basicSearch = (
+      <div className="va-flex va-flex--stretch msg-search-simple-wrap">
+        <ErrorableTextInput
+            field={this.props.params.term}
+            name="msg-search-simple"
+            label="Search messages"
+            onValueChange={this.handleSearchTermChange}/>
+        <button
+            type="submit"
+            className="msg-search-btn">
+          <i className="fa fa-search"></i>
+          <span className="msg-search-btn-text">Search</span>
+        </button>
+      </div>
+    );
+
+    let showBasicSearch;
+    if (!this.props.isAdvancedVisible) {
+      showBasicSearch = basicSearch;
```

Comment:
I meant to say that the JSX should be expressed only within the block, so this would be assigned the literal above.